server/sonar-server/src/main/java/org/sonar/server/app/ServerProcessLogging.java

```diff
@@ -52,12 +59,24 @@ public LoggerContext configure(Props props) {
     return ctx;
   }
 
-  private void configureAppenders(LoggerContext ctx, Props props) {
-    String logFormat = LOG_FORMAT.replace("XXXX", processName);
-    configureAppenders(logFormat, ctx, helper, props);
-  }
+  private void configureRootLogger(LoggerContext ctx, Props props) {
+    String logFormat = LOG_FORMAT.replace(THREAD_ID_PLACEHOLDER, threadIdField);
+    // configure appender
+    LogbackHelper.RollingPolicy rollingPolicy = helper.createRollingPolicy(ctx, props, processName);
+    FileAppender<ILoggingEvent> fileAppender = rollingPolicy.createAppender("file");
+    fileAppender.setContext(ctx);
+    PatternLayoutEncoder fileEncoder = new PatternLayoutEncoder();
+    fileEncoder.setContext(ctx);
+    fileEncoder.setPattern(logFormat);
+    fileEncoder.start();
+    fileAppender.setEncoder(fileEncoder);
+    fileAppender.start();
 
-  protected abstract void configureAppenders(String logFormat, LoggerContext ctx, LogbackHelper helper, Props props);
+    // configure logger
+    Logger rootLogger = ctx.getLogger(ch.qos.logback.classic.Logger.ROOT_LOGGER_NAME);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "rootLogger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this fully qualified name with "Logger" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1942)